apps/web/app/page.tsx

```diff
@@ -31,8 +38,10 @@ export default function Page() {
   return (
     <ThemeProvider theme={darkTheme}>
       <LayoutProvider>
-        <GlobalStyle />
-        <Layout />
+        <ModalProvider>
+          <GlobalStyle />
+          <Layout />
+        </ModalProvider>
```

Comment:
@MFelipeSilva why do we need a modal context?

Reply:
Why is it necessary to create a context that opens and closes the modal by the sign in and up button